tests/scripts/all.sh

```diff
@@ -1437,6 +1437,27 @@ component_test_tls1_2_default_cbc_legacy_cbc_etm_cipher_only_use_psa () {
     tests/ssl-opt.sh -f "TLS 1.2"
 }
 
+component_test_tls1_2_ecjpake_compatibility() {
```

Comment:
Can you also add a comment explaining the motivation of this test? This is something we've not been doing enough in the past and we're trying to improve. (In case you're wondering why I'm asking you to go beyond what you can see in the rest of this file.) Suggestion:

> We're not aware of any other (open source) implementation of EC J-PAKE in TLS that we could use for interop testing. However, we now have sort of two implementations ourselves: one using PSA, the other not. At least test that these two interoperate with each other.

(Note: if we were able to test interop with other implementations, we probably wouldn't bother to do this: we'd add tests of interop with the other implementation(s) in `ssl-opt.sh`, which is already run in multiple configs, both with and without `USE_PSA`, and that would be the end of it. But we can't, so here we are.)